File: frontend/src/models/Pret.ts
export interface PretBancaire {
  id: number;
  n_compte: string;
  nom_client: string;
  nom_banque: string;
  montant: number;
  date_pret: string; // YYYY-MM-DD
  taux_pret: number;
}

export interface PretBancaireCreate {
  n_compte: string;
  nom_client: string;
  nom_banque: string;
  montant: number;
  date_pret: string;
  taux_pret: number;
}

File: app_mobile/backend/app/schemas/pret.py
from pydantic import BaseModel
from datetime import date

class PretBancaireBase(BaseModel):
    n_compte: str
    nom_client: str
    nom_banque: str
    montant: float
    date_pret: date
    taux_pret: float

class PretBancaireCreate(PretBancaireBase):
    pass

class PretBancaireUpdate(PretBancaireBase):
    pass

class PretBancaire(PretBancaireBase):
    id: int

    class Config:
        from_attributes = True
